common/endian.h

```diff
@@ -411,8 +428,13 @@
 	#define TO_BE_64(a) SWAP_BYTES_64(a)
 	#define CONSTANT_LE_64(a) ((uint64)(a))
 	#define CONSTANT_BE_64(a) SWAP_CONSTANT_64(a)
+/** @} */
 
-// if the unaligned load and the byteswap take alot instructions its better to directly read and invert
+/** @name  Funtions for directly reading/writing and inverting
```

Comment:
There is a typo here. It also looks like two closing doxygen section braces in this file

Reply:
Right, the typo was here originally and I didn't catch it, thanks.

There are two closing tags at the end of the file because the first one closes this particular group (if it should be closes earlier, let me know), and the second one closes the main doxygen group for this file (introduced by defgroup at the beginning).

Current output of this file can be checked here: https://b-gent.github.io/PR2499/d8/d71/group__common__endian.html